SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -66,6 +68,8 @@
 
   @Inject
   public SingularityValidator(SingularityConfiguration configuration, DeployHistoryHelper deployHistoryHelper, RequestManager requestManager) {
+    this.configuration = configuration;
```

Comment:
I'd mimic what we're doing everywhere else in this class and just have a variable for the specific configuration value you need (in this case, `defaultBounceExpirationMinutes`).